jobs/override/base.py

```diff
@@ -0,0 +1,3 @@
+from jobs.settings import *
```

Comment:
Maybe will be better to start packaging `jobs/settings/` instead of define an override. I think this will be simpler

Reply:
Yes I agree, I didn't want to break existing environments, that's why I used a different name.. but if you're ok with it I will migrate to the settings folder

Reply:
For me is OK, what do you think @jbagot?

Reply:
Already done with the settings. I think with the new approach is better. Nice :)